### PR TITLE
System: Add choice console language in config.

### PR DESCRIPTION
**Pull request by Zangetsu38**
- Add choice language inside config file.
Tested with few langue full working, here in french
image
Ingame here in spanich
image
List of Language available, first is 0 and last is 19
>	Japanese = 0
	American English = 2 // Default value 
	French = 2
	Spanish = 3
	German = 4
	Italian = 5
	Dutch = 6
	Portugal Portuguese = 7
	Russian = 8
	Korean = 9
	Traditional Chinese = 10
	Simplified Chinese = 11
	Finnish = 12
	Swedish = 13
	Danish = 14
	Norwegian  = 15
	Polish = 16
	Brazil Portuguese = 17
	British English = 18
	Turkish = 19

so set in config 0 to 19 like you want used
- Important for user: All PSVita game not have all language, very lot gamer have only English.

### CODE REVIEW

File: src/emulator/modules/SceAppUtil/SceAppUtil.cpp [hunk 1]
```diff
@@ -273,9 +273,15 @@ EXPORT(int, sceAppUtilStoreBrowse) {
 }
 
 EXPORT(int, sceAppUtilSystemParamGetInt, unsigned int paramId, int *value) {
+    auto sys_lang = static_cast<SceSystemParamLang>(host.cfg.sys_lang);
```
Comment (pent0) [inline src/emulator/modules/SceAppUtil/SceAppUtil.cpp]:
```suggestion
    const SceSystemParamLang sys_lang = static_cast<SceSystemParamLang>(host.cfg.sys_lang);
```

File: src/emulator/modules/SceAppUtil/SceAppUtil.cpp [hunk 2]
```diff
@@ -273,9 +273,15 @@ EXPORT(int, sceAppUtilStoreBrowse) {
...
+    const auto default_sys_lang = SCE_SYSTEM_PARAM_LANG_ENGLISH_US;
+
+    if (host.cfg.sys_lang > static_cast<SceSystemParamLang>(19))
```
Comment (pent0) [inline src/emulator/modules/SceAppUtil/SceAppUtil.cpp]:
```suggestion
    if (host.cfg.sys_lang > SCE_SYSTEM_PARAM_LANG_TURKISH)
```

File: src/emulator/modules/SceAppUtil/SceAppUtil.cpp (hunk 1 above)
Comment (Zangetsu38) [reply src/emulator/modules/SceAppUtil/SceAppUtil.cpp]: const value can't works, i have already tested const auto, it require value editable
image

Comment (pent0) [reply src/emulator/modules/SceAppUtil/SceAppUtil.cpp]: Okayy, i forgot. Sorry :)

**Comment by Zangetsu38** [comment]
done changed

**Pull request merged**